packages/quantic/typings/lwc/coveoua.resource.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "@salesforce/resourceUrl/coveoua" {
```

Comment:
why do you need to push this .d.ts file?

Reply:
That files essentially tests TypeScript that `coveoua` is a global module.

It has nothing to do with the package, it's just to help the autocomplete in vscode essentially.